### PR TITLE
ValueError FAQ

### PR DESCRIPTION
**Pull request by oguzziya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +268,20 @@ python setup.py install --user
 ```
 </details>
 
+### ValueError while importing preCICE
+If you face with an error something like
```
Comment (BenjaminRodenberg) [inline README.md:272]:
```suggestion
If you face the error:
```

File: README.md [hunk 2]
```diff
@@ -268,6 +268,20 @@ python setup.py install --user
...
+
+```bash
+ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
+```
+
+make sure that you are using up-to-date version of NumPy. You can update NumPy with
```
Comment (BenjaminRodenberg) [inline README.md:278]:
```suggestion
make sure that you are using an up-to-date version of NumPy. You can update NumPy with
```

File: README.md [hunk 3]
```diff
@@ -268,6 +268,20 @@ python setup.py install --user
...
+
+```bash
+pip3 install numpy --upgrade
```
Comment (BenjaminRodenberg) [inline README.md:281]: Shouldn't we add the `--user` flag here? Or does `pip3` magically do the right thing when upgrading? Did you have to run this command as `sudo`?

**Review by BenjaminRodenberg — APPROVED** [review]
Generally looks good. Just some minor corrections and a question.

File: README.md (hunk 3 above)
Comment (oguzziya) [reply README.md:281]: My knowledge of pip is limited, unfortunately. When I checked it up I could not find any practices to follow. Adding`--user` would be the safe option I think. I did not use `sudo` while running it but in general I never had to use sudo while using pip.

Comment (BenjaminRodenberg) [reply README.md:281]: Then let's just keep it as it is. I think newer pip version apply the `--user` flag by default.

**Pull request merged**